agent/bench-scripts/pbench-trafficgen

```diff
@@ -100,7 +99,7 @@ sysinfo="none"
 tool_period="binary-search" # can also be "repeat-final-validation"
 trafficgen_version="unknown"
 if [[ -z "${trafficgen_dir}" ]]; then
-	trafficgen_dir="/opt/trafficgen"
+	trafficgen_dir="/opt/bench-trafficgen/trafficgen"
```

Comment:
Should that be `pbench-trafficgen` (with a leading `p`)?

Reply:
That is apparently when the "Crucible" environment calls it.